packages/api/portfolioDrafts/getPortfolioDrafts.ts

```diff
@@ -46,7 +55,9 @@ export async function handler(event: APIGatewayProxyEvent): Promise<APIGatewayPr
 
   try {
     const data = await client.send(command);
-    return new ApiSuccessResponse<PortfolioDraftSummary[]>(data.Items as PortfolioDraftSummary[], SuccessStatusCode.OK);
+    const items = (data.Items ?? []) as PortfolioDraftSummary[];
+    const summaries = items.map((draft) => exhaustivePick(draft, portfolioDraftFields));
```

Comment:
And this is how we can use the `pick` and now that we're going to have _all_ the required fields from the interface

Reply:
Looks good and glad we can apply this elsewhere as needed!